miqbox/miqbox.py

```diff
@@ -373,6 +373,30 @@ def start(connection, name):
     except Exception:
         click.echo("Fail to start appliance...")
 
+@cli.command(help="Restart Appliance Server")
+@click.argument("name")
+@connection
+def restart(connection, name):
```

Comment:
required:
 Please use different name `restart` just like pointing restart `appliance vm`. we are restarting `server`.

Reply:
- Used ```evmserver``` instead of restart.
- So now to execute restart command: ```miqbox evmserver --restart appliance_id_or_name```